docs/override.md

```diff
@@ -110,12 +110,36 @@ To show an informational alert box on top of the main content using the custom m
 
 ```json
```

Comment:
Should also state that each content element will become a paragraph in the alert box ...